types/proptest-regressions/network_address/mod.txt

```diff
@@ -1,7 +0,0 @@
-# Seeds for failure cases proptest has generated in the past. It is
```

Comment:
Can you add this to gitignore so that we don't commit this accidentally in future?